dev.geopjr.Turntable.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: dev.geopjr.Turntable
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+command: dev.geopjr.Turntable
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --talk-name=org.mpris.MediaPlayer2.*
+  - --talk-name=org.freedesktop.DBus
```

Comment:
why this dbus talk name?

Reply:
It's for listening [NameOwnerChanged](https://dbus.freedesktop.org/doc/dbus-java/api/org/freedesktop/DBus.NameOwnerChanged.html) and updating the available MPRIS buses.

Reply:
needs an exception